### PR TITLE
B28G10-87_Paha

### PR DESCRIPTION
**Pull request by Pahirdin**

### CODE REVIEW

File: src/test/java/com/canis/step_definitions/B28G10_87_Login_StepDefinitions.java [hunk 1]
```diff
@@ -0,0 +1,92 @@
...
+
+    @Given("user is on the login page")
+    public void user_is_on_the_login_page() {
+        Driver.getDriver().get(ConfigurationReader.getProperty("url"));
+    }
+
+    @When("user login as {string}")
+    public void user_login_as(String userType) {
+
+        String username = "marketing1@cybertekschool.com";
```
Comment (marick777) [inline src/test/java/com/canis/step_definitions/B28G10_87_Login_StepDefinitions.java]: What for we need line 23 and 24?? We using configuration.properties for credentials

File: src/test/java/com/canis/step_definitions/B28G10_87_Login_StepDefinitions.java [hunk 2]
```diff
@@ -0,0 +1,92 @@
...
+        Assert.assertTrue(actualMessage.equals(expectedMessage));
+
+    }
+
+
+    @Given("user is already logged in to CRM as {string}")
+    public void userIsAlreadyLoggedInToCRMAs(String userType) {
+
+
+        String username = "marketing1@cybertekschool.com";
```
Comment (marick777) [inline src/test/java/com/canis/step_definitions/B28G10_87_Login_StepDefinitions.java]: What for we need line 71 and 72?? We using configuration.properties for credentials

**Review by marick777 — CHANGES_REQUESTED** [review]
Check notes in src/test/java/com/canis/step_definitions/B28G10_87_Login_StepDefinitions.java

File: src/test/java/com/canis/step_definitions/B28G10_87_Login_StepDefinitions.java (hunk 1 above)
Comment (Pahirdin) [reply src/test/java/com/canis/step_definitions/B28G10_87_Login_StepDefinitions.java]: Ops that is a mistake. That was for the test. I will leave  username = ""    password=""

File: src/test/java/com/canis/step_definitions/B28G10_87_Login_StepDefinitions.java (hunk 2 above)
Comment (Pahirdin) [reply src/test/java/com/canis/step_definitions/B28G10_87_Login_StepDefinitions.java]: the same problem, I forgot to delate them.

**Review by marick777 — APPROVED** [review]
Looks good ;)

**Review by longnguyen1223 — APPROVED** [review]

**Pull request merged**